stable/index.yaml

```diff
@@ -352,12 +367,33 @@ entries:
     sources:
     - https://github.com/portworx/helm/tree/master/charts/px-backup
     urls:
-    - https://raw.githubusercontent.com/portworx/helm/master/stable/px-backup-1.0.0.tgz
+    - https://raw.githubusercontent.com/portworx/helm/2.2.1-dev/stable/px-backup-1.0.0.tgz
     version: 1.0.0
   px-central:
+  - apiVersion: v1
+    appVersion: 2.2.1-dev
+    created: "2022-05-30T20:00:44.119487+05:30"
+    description: A Helm chart for installing PX-Central on Kubernetes and Openshift
+    digest: c6b418ca8464e20e14e6607c7d2e0d257f884ffa69d1176293fe500728719efc
+    home: https://portworx.com/
+    icon: https://raw.githubusercontent.com/portworx/helm/master/doc/media/px-logo.png
+    keywords:
+    - px-backup
+    - storage
+    - persistent disk
+    - pvc
+    - portworx
+    - px-central
+    - lighthouse
+    name: px-central
+    sources:
+    - https://github.com/portworx/helm/tree/master/charts/px-central
+    urls:
+    - https://raw.githubusercontent.com/portworx/helm/2.2.1-dev/stable/px-central-2.2.1.tgz
+    version: 2.2.1
```

Comment:
@diptiranjanpx Is this expected ?. I am seeing a new entry along with the existing entry

Reply:
Yes this is expected.